aiohomekit/zeroconf.py

```diff
@@ -57,8 +59,13 @@
 
 logger = logging.getLogger(__name__)
 
+if sys.version_info[:2] < (3, 10):
```

Comment:
I am happy to follow HA versions here and drop < 3.10 to avoid the if, wdyt?

Reply:
That works for me. We could drop the asyncio.timeout/async_timeout fallback in utils.py as well

Reply:
https://github.com/Jc2k/aiohomekit/pull/325

Reply:
3.11 only like HA might too aggressive. not sure

Reply:
I think it probably is for anyone else using this.... is anything else using this?

Reply:
No one's told us if they are, and we know that the CLI pairing broke during the multi transport changes so they can't really be using that.